src/RegistrationController.js

```diff
@@ -198,6 +198,18 @@ export default BaseLoginController.extend({
         noCancelButton: true,
         title: loc('registration.form.title', 'login'),
         save: loc('registration.form.submit', 'login'),
+        initialize() {
+          this.listenTo(this.model, 'invalid error', (model, errorResp) => {
```

Comment:
are you able to modify in function `parseErrorMessage` below?

Reply:
`parseErrorMessage` is not triggered on form field validation

Reply:
@haishengwu-okta It's on-focus errors, not server errors. It should go to validate() function of the model. Can you suggest any better way to do this ?

Reply:
and by form field validation, I validating when an input form field loses focus

Reply:
oh I see. my bad. does that mean we totally ignore the message from backend?

Reply:
@haishengwu-okta that is not the intention, I think I should remove the `error` event listener, this should only change model validation messages